src/amberscript/parser.cc

```diff
@@ -1005,6 +1005,11 @@ Result Parser::ParseRun() {
     if (!pipeline->IsGraphics())
       return Result("RUN command requires graphics pipeline");
 
+    if (pipeline->GetVertexBuffers().size() > 1)
```

Comment:
nit {}'s

Reply:
Um.. what do you mean?

Reply:
The if() has a multi-line body, add {}'s around the body. Sorry, I realize I never explained myself, heh.